src/egon/data/datasets/gas_grid.py

```diff
@@ -1034,7 +1094,7 @@ class GasNodesAndPipes(Dataset):
     #:
     version: str = "0.0.9"
 
-    tasks = (insert_gas_data,)

```

Comment:
Is the function` insert_gas_data `still used somewhere?

Reply:
It was used to generate the gas grid for one scenario that will be copied for other scenarios. It is not necessary for the status2019 but it is for future cases.

Reply:
This problem was already corrected in 489eebeed1513adf8cb08570803cbe2f301566da

Reply:
Okay, since you did this change after my first checks last week, I couldn't see it.